teacher.rb

```diff
@@ -0,0 +1,16 @@
+require_relative './person_class'
+
+class Student < Person
+  def intialize(classroom)
```

Comment:
- Let's update the constructor for the class Teacher as you've made this little mistake using the details for the Student and also for the Teacher. The teacher has a specialization. the constructor should be this below:
` def initialize(age, specialization, name = 'Unknown')`